src/module/hook/timeline/index.ts

```diff
@@ -41,13 +42,13 @@ export function useTimelinesByVentureId(
 export function useTimelinesByUserId(
   searchTimelinesByUserId: ISearchTimelinesbyUserId
 ) {
-  return useQuery<any, ErrorResponse>(
+  const enabled = Boolean(
+    searchTimelinesByUserId.token && searchTimelinesByUserId.userId
+  );
+  return useQuery<ITimeline[], ErrorResponse>(
```

Comment:
I'll be replacing `any` with real types as I come across them. No need to do all of them up front though.